jerry-core/ecma/base/ecma-helpers.cpp

```diff
@@ -800,10 +800,24 @@ ecma_free_internal_property (ecma_property_t *property_p) /**< the property */
     case ECMA_INTERNAL_PROPERTY_EXTENSION_ID: /* an integer */
     case ECMA_INTERNAL_PROPERTY_NON_INSTANTIATED_BUILT_IN_MASK_0_31: /* an integer (bit-mask) */
     case ECMA_INTERNAL_PROPERTY_NON_INSTANTIATED_BUILT_IN_MASK_32_63: /* an integer (bit-mask) */
+    case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_TARGET_FUNCTION:
+    case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_BOUND_THIS:
     {
       break;
     }
 
+    case ECMA_INTERNAL_PROPERTY_BOUND_FUNCTION_BOUND_ARGS:
+    {
+      if (property_value != ECMA_NULL_POINTER)
+      {
+        ecma_free_values_collection (ECMA_GET_NON_NULL_POINTER (ecma_collection_header_t,
+                                                                property_value),
+                                     false);
```

Comment:
This format is used in [L745](https://github.com/Samsung/jerryscript/blob/master/jerry-core/ecma/base/ecma-helpers.cpp#L745) and [L756](https://github.com/Samsung/jerryscript/blob/master/jerry-core/ecma/base/ecma-helpers.cpp#L756).
What would be the appropriate?